grpc-ballerina/tests/22_retry_service.bal

```diff
@@ -37,11 +37,61 @@ service "RetryService" on retryListener {
         requestCount += 1;
         io:println(clientName + ": Attetmpt No. " + requestCount.toString());
```

Comment:
Shall we use `log:printInfo()` since this is inside a service.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/219/commits/2a44acac6a26d011708bf186f99566cc19fa06e4